nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/uidgen/UIDSequencer.java

```diff
@@ -60,6 +67,7 @@
      *
      * @param key
      * @return
+     * @deprecated since 9.10 use {@link #getNextLong(String)} instead.
      */
     int getNext(String key);
 
```

Comment:
same as above, annotation missing

Reply:
Furthermore, in `AbstractUIDSequencer`, you should remove `getNextLong` and provide an implementation of `getNext` which calls `getNextLong`. No need to put it `abstract`, this method is delcared in the interface.
By doing that you should probably change `JPAUIDSequencerImpl#getNext` by `JPAUIDSequencerImpl#getNextLong`.

Reply:
At the same time could you clean the class ? Remove un-needed javadoc tags, un-needed abstract method declared in interface... ?